src/SFA.DAS.EmployerApprenticeshipsService.Application/Services/ApprenticeshipInfoServiceWrapper.cs

```diff
@@ -73,7 +75,7 @@ public ProvidersView GetProvider(long ukPrn)
                 throw;
             }
             catch (EntityNotFoundException)
-            {
+            {//Put in check here, select provider name from PaymentMetaData join Payment on paymentmetadataid where ukprn = this.ukprn
```

Comment:
Does this need doing or can the comment be deleted??

Reply:
Note for myself, will remove